composer.json

```diff
@@ -43,9 +43,9 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "0.5-dev"
+            "dev-master": "1.1-dev"
```

Comment:
thinking about this again, i think the main argument for calling this 1.0 is that people can get their fixes by depending on `1.0.*@dev` and play it safe with not ending up with new features of 1.1 that might have side effects for them.